t/29_sdaf_inventory_tools.t

```diff
@@ -9,6 +9,57 @@ use testapi;
 use sles4sap::sap_deployment_automation_framework::inventory_tools;
 use Data::Dumper;
 
+# SDAF inventory data example:
+# - Single host group
+# - Group with multiple hosts
+# - Empty group
+
+my $mock_inventory_data = {
```

Comment:
Is it the bare minimal structure to make all tests to pass?

Reply:
I would say so. 
Bare minimal structure a function must be able to handle.